homeassistant/components/drop_connect/sensor.py

```diff
@@ -0,0 +1,314 @@
+    ),
+    DROPSensorEntityDescription(
+        key=PEAK_FLOW_RATE,
+        translation_key=PEAK_FLOW_RATE,
+        icon="mdi:shower-head",
+        native_unit_of_measurement="gpm",
+        suggested_display_precision=1,
+        value_fn=lambda device: device.peak_flow_rate,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    DROPSensorEntityDescription(
+        key=WATER_USED_TODAY,
+        translation_key=WATER_USED_TODAY,
+        device_class=SensorDeviceClass.WATER,
+        native_unit_of_measurement=UnitOfVolume.GALLONS,
+        suggested_display_precision=1,
+        value_fn=lambda device: device.water_used_today,
+    ),
+    DROPSensorEntityDescription(
+        key=AVERAGE_WATER_USED,
+        translation_key=AVERAGE_WATER_USED,
+        device_class=SensorDeviceClass.WATER,
+        native_unit_of_measurement=UnitOfVolume.GALLONS,
+        suggested_display_precision=0,
+        value_fn=lambda device: device.average_water_used,
+    ),
+    DROPSensorEntityDescription(
+        key=CAPACITY_REMAINING,
+        translation_key=CAPACITY_REMAINING,
+        device_class=SensorDeviceClass.WATER,
+        native_unit_of_measurement=UnitOfVolume.GALLONS,
+        suggested_display_precision=0,
+        value_fn=lambda device: device.capacity_remaining,
+    ),
+    DROPSensorEntityDescription(
+        key=CURRENT_SYSTEM_PRESSURE,
+        translation_key=CURRENT_SYSTEM_PRESSURE,
+        device_class=SensorDeviceClass.PRESSURE,
+        native_unit_of_measurement=UnitOfPressure.PSI,
+        suggested_display_precision=1,
+        value_fn=lambda device: device.current_system_pressure,
+    ),
+    DROPSensorEntityDescription(
+        key=HIGH_SYSTEM_PRESSURE,
+        translation_key=HIGH_SYSTEM_PRESSURE,
+        device_class=SensorDeviceClass.PRESSURE,
+        native_unit_of_measurement=UnitOfPressure.PSI,
+        suggested_display_precision=0,
+        value_fn=lambda device: device.high_system_pressure,
+    ),
+    DROPSensorEntityDescription(
+        key=LOW_SYSTEM_PRESSURE,
+        translation_key=LOW_SYSTEM_PRESSURE,
+        device_class=SensorDeviceClass.PRESSURE,
+        native_unit_of_measurement=UnitOfPressure.PSI,
+        suggested_display_precision=0,
+        value_fn=lambda device: device.low_system_pressure,
+    ),
+    DROPSensorEntityDescription(
+        key=BATTERY,
+        device_class=SensorDeviceClass.BATTERY,
+        native_unit_of_measurement=PERCENTAGE,
+        suggested_display_precision=0,
+        value_fn=lambda device: device.battery,
+    ),
+    DROPSensorEntityDescription(
+        key=TEMPERATURE,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        native_unit_of_measurement="°F",
+        suggested_display_precision=1,
+        value_fn=lambda device: device.temperature,
+    ),
+    DROPSensorEntityDescription(
+        key=INLET_TDS,
+        translation_key=INLET_TDS,
+        icon=TDS_ICON,
+        native_unit_of_measurement="ppm",
```

Comment:
use `CONCENTRATION_PARTS_PER_MILLION` from `homeassistant.const`

Reply:
Fixed in 384620ca853e5d8a6d226bed3e26b60c678a8efd.